Trino Summit Teaser post (#351)
trinodb/trino.io: _posts/2022-09-16-trino-summit-2022-teaser.md

---
layout: post
title: "Trino Summit 2022 will be cute and legendary"
author: "Brian Olsen"
excerpt_separator: <!--more-->
image: /assets/blog/trino-tenth-birthday/creators.jpeg
---

> Bun Bun is back and this year we have an exciting lineup of speakers. The range

Reviewer: ```suggestion
Commander Bun Bun is back, and this year we have an exciting lineup of speakers. The range
```

> of topics span from architectures like data mesh and data lakehouses, to running

Reviewer: ```suggestion
of topics spans from architectures like data mesh and data lakehouses, to running
```

> Trino at scale with fault-tolerance and how to use `EXPLAIN`. This conference is

Reviewer: ```suggestion
Trino at scale with fault-tolerant execution and how to use `EXPLAIN`. This conference is
```

maybe even break up the sentence

> free and takes place on November 10th. The summit is in-person but also is

Reviewer: The summit is a hybrid even for in-person and virtual attendance.

available to virtual attendees as well. Find out more details below!

<!--more-->

## Location

This years Trino Summit will be hosted live at the Commonwealth Club in San 
Francisco, CA. In-person registration is limited to 250 seats so make sure you
register quickly before spots run out!

> 

Reviewer: Add the button from the other blog post here

## Announcing the first sessions and speakers

> 1. Keynote: State of Trino

Reviewer: Add an entry sentence that mentions that call for speakers closed and we have a full line up .. and here are a first few teasers

    Description: Hear the latest on the state of the open source Trino project.
    Trino is the award-winning MPP SQL query engine. In this session, Trino and
    Presto creators will discuss the latest features that have landed in the
    last year, the roadmap for the year ahead, and community growth highlights.

>     * <img width="150" src="https://sessionize.com/image/40f9-0o0o0-Jd9Ri1YsJ7VyFniPcENpHv.jpg">

Reviewer: Let not use images from sessionize .. either dont use them .. or get them into the site directly

Reviewer: Just the names are perfectly fine as well btw. .. images are not really necessary
